Lab1/Client/CLL/CLL/Program.cs

```diff
@@ -0,0 +1,123 @@
+﻿using System;
+using System.Net;
+using System.Net.Sockets;
+using System.Text;
+using System.Threading;
+
+namespace CLL
+{
+    class Program
+    {
+
+
+        static void Main(string[] args)
+        {
+            IPAddress ip = IPAddress.Parse("127.0.0.1");
+            int port = 5000;
+            TcpClient client = new TcpClient();
+            client.Connect(ip, port);
+            Console.WriteLine("client connected!!");
+            NetworkStream ns = client.GetStream();
+            Thread thread = new Thread(o => ReceiveData((TcpClient)o));
+            thread.Start(client);
+            string s;
+
+
+
+            string name = string.Empty;
+            Console.ReadKey();
+            while (!chat)
+            //while (string.IsNullOrEmpty(name))
+            {
+                Console.WriteLine();
+                Console.WriteLine("Dont use symbol |");
+                Console.Write("Your name: ");
+                if (chat)
+                {
+                    break;
+                }
+                name = Console.ReadLine();
+                if (string.IsNullOrEmpty(name))
+                {
+                    continue;
+                }
+                Console.WriteLine("Dont use symbol |");
+                Console.Write("Your password: ");
+                string passw = Console.ReadLine();
+
+                if (name.IndexOf('|') != -1 || passw.IndexOf('|') != -1)
+                {
+                    Console.WriteLine("Found |");
+                    continue;
+                }
+
+                string ss = string.Concat(name, "|", passw);
+                byte[] logs = Encoding.ASCII.GetBytes(ss);
+                ns.Write(logs, 0, logs.Length);
+                Thread.Sleep(3000);
+            }
+            name = string.Concat(name, ": ");
+
+
+
+
+
+
+
+
+            while (!string.IsNullOrEmpty((s = Console.ReadLine())))
+            {
+                s = string.Concat(name, s);
+                byte[] buffer = Encoding.ASCII.GetBytes(s);
+                ns.Write(buffer, 0, buffer.Length);
```

Comment:
What if login and password length more than 1024 bytes?

Reply:
Server down/

Reply:
I assume u know what to do =)

Reply:
Made.